projects/demo/src/modules/app/pages.ts

```diff
@@ -936,6 +936,13 @@ export const pages: TuiDocPages = [
                 keywords: `editor, custom color picker, color picker, tool, wysiwyg, редактор, выбрать цвет, html, rich, text`,
                 route: `/editor/custom-tool/color-picker`,
             },
+            {
+                section: `Editor`,
+                title: `Font size tool`,
+                keywords: `editor, custom font size tool, tool, wysiwyg, редактор, выбрать размер, html, rich, text`,
+                route: `/editor/font`,
+                fragment: `custom-size-tool`,
```

Comment:
It does not actually works

https://github.com/Tinkoff/taiga-ui/assets/35179038/10aed5ac-5f5f-445c-8ee7-105a5228e266

Reply:
<img width="302" alt="image" src="https://github.com/Tinkoff/taiga-ui/assets/12021443/e65a592e-21ba-437e-b326-cca89a535791">

Reply:
fixed